### ISSUE TITLE
Mapping for prescribing rx_basis

### ISSUE THREAD

**Issue by bos666** [kumc-bmi/i2p-transform#30]
In pcornet_mapping.csv, are these supposed to be mapped to '\PCORI_MOD\RX_BASIS\PR\OT'?

pcori_path,local_path
\PCORI_MOD\RX_BASIS\PR\,\Medication\Inpatient\
\PCORI_MOD\RX_BASIS\PR\,\Medication\Other Orders\
\PCORI_MOD\RX_BASIS\PR\,\Medication\Outpatient\
\PCORI_MOD\RX_BASIS\PR\,\Medication\PRN Inpatient Order\

**Comment by dckc** [comment]
@njgraham did you see this issue?

**Comment by njgraham** [comment]
@dckc Yes, I see (finally).  I've been working on CDM medications lately so much of it is fresh in my mind - I'll hopefully look more in depth soon.

**Comment by dckc** [comment]
fixed in fbbe59b8daed7567f0534329275ea972a2d3fc43
